composer.json

```diff
@@ -47,6 +47,7 @@
     "require-dev": {
         "phpunit/phpunit": ">=5.0",
         "orchestra/testbench": "^3.0",
+        "mockery/mockery": "^1.0",
```

Comment:
Do we need this? Doesn't seem to be used on the diff

Reply:
The tests failed for PHP 5.6 unless I required it as a dev dependency, which resolved the issue. 

I think in the context of the refactor you've suggested below, I shouldn't need it anymore anyway.

See: https://github.com/ukfast/laravel-health-check/runs/929533806?check_suite_focus=true

Reply:
Facades have built-in mockery support so I'm not sure why it didn't work in PHP 5.6 by default.

The `shouldReceive` method in the cache test requires mockery.

https://github.com/laravel/framework/blob/master/src/Illuminate/Support/Facades/Facade.php#L82

Reply:
Fair enough. Just checking